.circleci/config.yml

```diff
@@ -265,6 +265,128 @@ commands:
             - ccache-cache-{{ arch }}-<< parameters.product >>-{{ .Branch }}
             - ccache-cache-{{ arch }}-<< parameters.product >>-
 
+  install-doc-deps:
+    description: Install dependencies needed to build the documentation
+    steps:
+      - run:
+          name: Install dependencies
+          command: |
+            apt-get update && apt-get -qq -y install \
+            autoconf \
+            automake \
+            bison \
+            curl \
+            flex \
+            g++ \
+            git \
+            latexmk \
+            libboost-all-dev \
+            libedit-dev \
+            libluajit-5.1-dev \
+            libssl-dev \
+            make \
+            pkg-config \
+            ragel \
+            virtualenv
+            if [ "${CIRCLE_PROJECT_USERNAME}" = "PowerDNS" -a "${CIRCLE_PROJECT_REPONAME}" = "pdns" -a "${CIRCLE_BRANCH}" = "master" ]; then
```

Comment:
This long line is repeated a few times - if we ever need to change it, it would be easy to make mistakes. Perhaps we can make it an alias or a script?

Reply:
Looks like [conditional steps](https://circleci.com/docs/2.0/reusing-config/#defining-conditional-steps) don't do what we want. I'll have a look at other solutions